qiskit_ibm_runtime/utils/noise_learner_result.py

```diff
@@ -139,20 +139,28 @@ class LayerError:
     Args:
         circuit: A circuit whose noise has been learnt.
         qubits: The labels of the qubits in the ``circuit``.
-        error: The Pauli Lindblad error channel affecting the ``circuit``.
+        error: The Pauli Lindblad error channel affecting the ``circuit``, or ``None`` if the error
+            channel is unknown.
 
     Raises:
         ValueError: If ``circuit``, ``qubits``, and ``error`` have mismatching number of qubits.
     """
 
     def __init__(
-        self, circuit: QuantumCircuit, qubits: Sequence[int], error: PauliLindbladError
+        self,
+        circuit: QuantumCircuit,
+        qubits: Sequence[int],
+        error: Optional[PauliLindbladError] = None,
```

Comment:
I cant see from the diff, but is there a doc string that needs updating to say what a value of None correponds to?

Reply:
Yes:
```
error: The Pauli Lindblad error channel affecting the ``circuit``, or ``None`` if the error
            channel is either unknown or explicitly disabled.
```